server/src/db/config.json

```diff
@@ -1,22 +1,22 @@
 {
   "development": {
-    "username": "jasonephraim",
+    "username": "mcraepeavy",
     "password": null,
     "database": "rigex-project",
     "host": "127.0.0.1",
     "dialect": "postgres",
     "port": 5432
   },
   "test": {
-    "username": "jasonephraim",
+    "username": "mcraepeavy",
```

Comment:
please make notes in readme about any env / setup additions

Reply:
refactoring config